src/aura/BGE_DonationSelector/BGE_DonationSelector.cmp

```diff
@@ -42,6 +42,7 @@
     <aura:attribute name="unpaidPayments" type="List" description="List of unpaid Payments" access="global" />
     <aura:attribute name="openOpportunities" type="List" description="List of open Opportunities" access="global" />
     <aura:attribute name="labels" type="Map" description="Labels retrieved in Apex related to Schema information." access="global"/>
+    <aura:attribute name="isPaymentCreationEnabled" type="Boolean" description="True if Automatic Payment Creation is enabled." access="public" default="false" />
```

Comment:
@lparrott - just want to call your attention to this new attribute in case it causes any concern. You were using this component for GEM, right?

Reply:
We sure are, this change shouldn't be an issue though, thanks for pointing it out!